game/core/src/me/lihq/game/models/Room.java

```diff
@@ -70,6 +70,11 @@
     private float animationStateTime = 0f;
 
     /**
+     * This list stores the coordinates of all hideable slots in this room
+     */
+    public List<Vector2Int> hideableTiles = new ArrayList<Vector2Int>();
```

Comment:
a better way would be to use the getter still (make this private, then in getHidingSpots() check to see if this is null and if so then it sets it otherwise it returns this